README.md

```diff
@@ -2,4 +2,4 @@
 GitHub for Developers
 
 ## Adding a header two
```

Comment:
@shapeous please remove the header two and we will be all set.

Reply:
Done @lcdezines!